src/kOS/Execution/CPU.cs

```diff
@@ -238,7 +238,7 @@ public ProgramContext GetInterpreterContext()
             return contexts[0];
         }
         
-        public ProgramContext GetProgramContext()
+        public ProgramContext SwitchToProgramContext()
```

Comment:
Maybe.  I wanted to hurry up and get the bug fixed.  The rename was a good first step that doesn't feel like doing a refactor but at least "un-hides" the effect the method has.